src/resources/views/crud/inc/form_save_buttons.blade.php

```diff
@@ -22,3 +22,26 @@
 
     <a href="{{ $crud->hasAccess('list') ? url($crud->route) : url()->previous() }}" class="btn btn-default"><span class="fa fa-ban"></span> &nbsp;{{ trans('backpack::crud.cancel') }}</a>
 </div>
+
+@push('after_scripts')
+<script>
+
```

Comment:
Usually it's best to have at least one comment for each JS code block - so that it's fast & easy to understand what that code block does without having to go through it. I'll do it.

Reply:
Done.